nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
+---
+
+The [distributor](/nservicebus/scalability-and-ha/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) with NServiceBus Version 6.
+
+## Client-side distribution
+
+With sender-side distribution, senders are aware of scaled out receivers, allowing senders to distribute messages directly to the receivers instead of using an intermediary distributor.
+
+
+## Upgrade scenario
+
+This process aims to allow upgrade in production environments without missing events and minimal downtime.
+
+WARNING: To avoid duplicate events, the Distributor node needs to be upgraded to a worker node using NServiceBus 6 (it can be decommissioned after all workers are upgraded to Version 6).
```

Comment:
the issue are remaining subscriptions.
* If you take down the distributor, you need to unsubscribe it first, which is quite tricky and you're facing a time where no subscription may be active which results in message loss
* If you upgrade a worker before shutting down the distributor you may get duplicate events (one for the v6 endpoint, one for the distributor)

Therefore the approach is to migrate the distributor to a worker node, so that the subscriptions in the publishers remain valid. The goal was to avoid "decommissioning steps" out of this guide (because it would require manually fiddling with the subscription storages which I really don't want to include in this guide since this alone is a dedicated doco page)